### PR TITLE
Improve buildscripts

### PR DESCRIPTION
**Pull request by PancakeTAS**
me when there's two dependency blocks specifying the same dependency with different versions 🪦

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -26,88 +25,54 @@ repositories {
...
-dependencies {
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.1'
-}
-
-tasks.named('test', Test) {
-    useJUnitPlatform()
-
-	testLogging {
-		events "passed", "skipped", "failed"
-	}
```
Comment (ScribbleTAS) [inline build.gradle:112]: You disabled unit testing by removing this... readd the testing block

**Review by ScribbleTAS — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (PancakeTAS) [reply build.gradle:112]: are you sure? unit testing still works for me

Comment (ScribbleTAS) [reply build.gradle:112]: You disabled unit testing when running build... There was no test report generated https://github.com/MinecraftTAS/TASmod/actions/runs/5058314731/jobs/9078263968#step:6:1

Comment (PancakeTAS) [reply build.gradle:112]: I see interesting, so eclipse uses a differnent system than gradle

**Review by ScribbleTAS — APPROVED** [review]

**Pull request merged**